packages/Card/stories/examples/BasicCardContent.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import Card from "../../src";
+
+export default () => {
+  const card = [
+    {
+      title: "Master view",
+      attrTypes: "25",
+      questions: "100",
+      creator: "Cory McBain",
```

Comment:
When the title, creator, or updater are longer than one line, it doesn't handle it gracefully.  Perhaps it should add `...`

Reply:
Maybe we can add this in next iteration? what do you think?

Reply:
Sure, I won't lose sleep over it :)